mex/common/organigram/extract.py

```diff
@@ -51,18 +52,28 @@ def get_unit_merged_ids_by_synonyms(
     """Return a mapping from unit alt_label and label to their merged IDs.
 
     There will be multiple entries per unit mapping to the same merged ID.
+    But if the same entry mapps to different merged IDs, an error is thrown.
```

Comment:
```suggestion
    But if the same entry maps to different merged IDs, an error is thrown.
```

this could also go into a `Raises: ...` block in the docstring